.castor/docker.php

```diff
@@ -32,9 +34,11 @@ function about(): void
     io()->section('Available URLs for this project:');
     $urls = [variable('root_domain'), ...variable('extra_domains')];
 
-    $payload = @file_get_contents(sprintf('http://%s:8080/api/http/routers', variable('root_domain')));
-    if ($payload) {
-        $routers = json_decode($payload, true);
+    try {
+        $routers = http_client()
+            ->request('GET', 'http://127.0.0.1:8080/api/http/routers')
```

Comment:
it was root_domain before, why this change?

Reply:
because it's useless to use the root domain. It's simpler like that, isn't?

Reply:
hmm, I don't know how docker machine works, but I have rollback this change